### PR TITLE
fix docs optimizer

### PR DESCRIPTION
**Pull request by sljlp**
### PR types

Others
### PR changes

Docs
### Describe

fix docs for optimizer
out.backward somewhere is incorrect
we change it to loss.backward

### CODE REVIEW

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by wangxicoding — APPROVED** [review]
